.travis.yml

```diff
@@ -8,17 +8,17 @@ matrix:
     osx_image: xcode10.3
     env:
       - MATRIX_EVAL="CC=clang && CXX=clang++ && PYTHON_VERSION=3.6.1 && PYTHON=python"
-      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip && LLVM_FILE=taichi-llvm-8.0.1.zip"
+      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip"
   - os: osx
     osx_image: xcode10.3
     env:
       - MATRIX_EVAL="CC=clang && CXX=clang++ && PYTHON_VERSION=3.7.1 && PYTHON=python"
-      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip && LLVM_FILE=taichi-llvm-8.0.1.zip"
+      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip"
```

Comment:
```suggestion
      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm10/taichi-llvm-10.0.0.zip"
```

Reply:
That's not going to work, starting from LLVM 10, all prebuilt binaries will be hosted in taichi-dev/taichi_assets instead of yuanming's personal account.